src/main/res/values/strings.xml

```diff
@@ -19,13 +19,13 @@
 
 <resources>
 
-    <string name="application_name" translatable="false">Amahi</string>
+    <string name="application_name">Amahi</string>
 
 
     <string name="title_apps">Apps</string>
     <string name="title_settings">Settings</string>
     <string name="title_shares">Shares</string>
-    <string name="title_selected_server">Server</string>
+    <string name="title_selected_server">selected server</string>
```

Comment:
Is there a reason this became lowercase? Seems like a title, so it ought to be "Selected Server" or ... why not just "Server" (less words)?

Reply:
?